app/assets/stylesheets/application.css.scss

```diff
@@ -91,9 +96,49 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
   }
 
   body {
+    background-color: $bg-color;
     margin: 0 0 20px;
   }
 
+  form {
+    background: $white;
+    border: 1px solid $border;
+    border-collapse: inherit; 
+    border-radius: 5px;
+  }
+
+  #login-box form {
+    border:none;
+    border-radius: none;
+    margin: 0;
+    padding: 0;   
+  }
+
+  form {
+    margin-bottom:20px; 
+    padding: 20px;
+    
+  #user_notifications {
```

Comment:
Avoid using id selectors<br>Line should be indented 4 spaces, but was indented 2 spaces<br>Selector `user_notifications` should be written in lowercase with hyphens